<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowImportXml</class>
 <widget class="QDialog" name="windowImportXml">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,10,0,0,0,0" columnstretch="1,0,0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblProgressStatus">
     <property name="text">
      <string>Импорт XML-файлов</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblCurrentFilename">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="lblReport">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblFilenameLabel">
     <property name="text">
      <string>Текущий файл:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="2" colspan="2">
    <widget class="QToolButton" name="tbuttonOK">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QProgressBar" name="pbarImportProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
